### ISSUE TITLE
Unavailiable clusters after 2.4.4 -> 2.4.5 Upgrade

### ISSUE THREAD

**Issue by pancochea** [rancher/rancher#27809]
**What kind of request is this (question/bug/enhancement/feature request):**
Bug

**Steps to reproduce (least amount of steps as possible):**
Create a Cluster with rancher 2.4.4 over Kubernetes v1.17.6-rancher2-2.

Create a cluster of any kind.

Update from 2.4.4 to 2.4.5

**Result:**
Clusters are not availiable, only nodes information is accessible and rancher looses control over the managed cluster.

**Other details that may be helpful:**
Tried a Amazon provider cluster, none provider cluster, a single machine cluster (etc. workers and controlplane in a single instance) and other convinations with same results.

**Environment information**
- Rancher version: 2.4.4 to 2.4.5
- Installation option (single install/HA): HA

**Cluster information**
- Cluster type (Hosted/Infrastructure Provider/Custom/Imported): Imported cluster seems to be workings

- Kubernetes version (use `kubectl version`): v1.17.6

- Docker version (use `docker version`): 1.19
Aditional image. All clujsters where up and running before the upgrade, the image is just after the upgrade.

![image (1)](https://user-images.githubusercontent.com/54702555/86368272-4737e200-bc7d-11ea-9ced-c48f0fba5ad2.png)

**Comment by jiaqiluo** [comment]
Hi @pancochea  Is your setup recovered eventually? 

I cannot reproduce the error. All clusters are active and working well after upgrading Rancher. 

<img width="1479" alt="Screen Shot 2020-07-02 at 10 25 27 AM" src="https://user-images.githubusercontent.com/6218999/86392597-9582bc00-bc50-11ea-86e7-ea22a4bf43b2.png">

here is what I did:
- user RKE `v1.1.2` to provision a 3-node cluster will all roles in each node
- install Rancher `v2.4.4`
- log in as admin 
- provision the following clusters: 
  - 1 etcd, 1 control plane, 3 workers, EC2 
  - 1 etcd + control plane, 4 workers, EC2
  - 1 etcd, 1 control plane, 3 workers, Digital Ocean
  - 1 etcd, 1 control plane, 3 workers, EC2, imported cluster
- enable cluster monitoring on all those clusters 
- upgrade rancher to v2.4.5

**Comment by pancochea** [comment]
We used RKE 1.0.10 to create that specific cluster. That cluster is the one we use as a sandbox, and testing (like testing updates). We re-recreated the whole rancher with RKE 1.0.10 and kubernetes v1.17.6-rancher2-2.

From the update I understand that you tried **without** monitoring? Our clusters don't have monitoring so that will be closer to our set up.

We did the whole process twice with the same result, so there is somethin we are missing.

As soon as I can I'll try to reproduce your steps.
